quinn-proto/Cargo.toml

```diff
@@ -23,8 +23,10 @@ certificate-transparency = ["ct-logs"]
 tls-rustls = ["rustls", "webpki", "ring"]
 # Trust the contents of the OS certificate store by default
 native-certs = ["rustls-native-certs"]
+unstructured-data = ["arbitrary"]
```

Comment:
What's this feature for?

Reply:
Ah, well this is me still misunderstanding above re: setting arbitrary behind a feature :-) 
https://github.com/djc/quinn/pull/831#discussion_r467640018. I'll take this to the chat to sort out

Reply:
I took a try at moving the `arbitrary` as an optional dependency and dropping this feature, if I am understanding above correctly.

Reply:
LGTM!